### PR TITLE
Use ol.inherits instead of goog.inherits

### PR DESCRIPTION
**Pull request by fredj**

### CODE REVIEW

File: contribs/gmf/apps/desktop_alt/js/controller.js [hunk 1]
```diff
@@ -33,15 +33,14 @@ gmf.module.value('ngeoQueryOptions', {
...
-  goog.base(
-      this, {
-        srid: 21781,
-        mapViewConfig: {
-          center: [632464, 185457],
-          zoom: 3,
-          resolutions: [250, 100, 50, 20, 10, 5, 2, 1, 0.5, 0.25, 0.1, 0.05]
-        }
-      }, $scope, $injector);
+  gmf.AbstractDesktopController(this, {
```
Comment (adube) [inline contribs/gmf/apps/desktop_alt/js/controller.js]: gmf.AbstractDesktopController.call

Comment (fredj) [inline contribs/gmf/apps/desktop_alt/js/controller.js]: thanks, fixed

**Comment by adube** [comment]
LGTM, +1.

**Comment by fredj** [comment]
to be merged after the demo (29/08/2016)

**Pull request merged**